compose-builder/Makefile

```diff
@@ -520,6 +520,37 @@ ifeq (true, $(DS_GROVE))
  endif
 endif
 
+COMPOSE_DOWN:=  DEV= \
```

Comment:
Same here. Change `:=` to `=`. Alternatively you can use a [define block](https://www.gnu.org/software/make/manual/html_node/Multi_002dLine.html)

```make
define COMPOSE_DOWN
... code here ...
endef
```

Reply:
I like it. Thanks!